### ISSUE TITLE
add citation block generated from TEI metadata to all content types, including license where appropriate

### ISSUE THREAD

**Issue by dasch124** [acdh-oeaw/vicav-vue3#148]

**Comment by VeronikaEngler** [comment]
Veronika and  I would propose a structure like this:
Ritt-Benmimoun, Veronika (ed.) 2024. _TUNOCENT- Tunisia’s Linguistic terra incognita: An Investigation into the Arabic Varieties of Northwestern and Central Tunisia_. (Available online at https://tunocent-5b7ead81-4e64-4879-83de-7120e271b4f4.acdh-ch-dev.oeaw.ac.at/?w=W10=&a=smart-tile, Accessed on 2024-09-11.)

for the website in general and a structure like these:
Schwemmer, Franziska. 2024. "Profile of Redeyef Arabic". In: Ritt-Benmimoun, Veronika (ed.) 2024. _TUNOCENT- Tunisia’s Linguistic terra incognita: An Investigation into the Arabic Varieties of Northwestern and Central Tunisia_. (Available online at https://tunocent-5b7ead81-4e64-4879-83de-7120e271b4f4.acdh-ch-dev.oeaw.ac.at/?w=W10=&a=smart-tile, Accessed on 2024-09-11.)
or 
Aloui, Nidhal; Naddari, Aleksandra & Ritt-Benmimoun, Veronika. 2024. "A Sample of Gboullat Arabic". In: Ritt-Benmimoun, Veronika (ed.) 2024. _TUNOCENT- Tunisia’s Linguistic terra incognita: An Investigation into the Arabic Varieties of Northwestern and Central Tunisia_. (Available online at https://tunocent-5b7ead81-4e64-4879-83de-7120e271b4f4.acdh-ch-dev.oeaw.ac.at/?w=W10=&a=smart-tile, Accessed on 2024-09-11.)
or
Ercegovčević, Aleksandra & Ritt-Benmimoun, Veronika. 2024. "Linguistic Features of Bou Saada Arabic". In: Ritt-Benmimoun, Veronika (ed.) 2024. _TUNOCENT- Tunisia’s Linguistic terra incognita: An Investigation into the Arabic Varieties of Northwestern and Central Tunisia_. (Available online at https://tunocent-5b7ead81-4e64-4879-83de-7120e271b4f4.acdh-ch-dev.oeaw.ac.at/?w=W10=&a=smart-tile, Accessed on 2024-09-11.)

for specific parts/contributions.

**Comment by MauPalantir** [comment]
@VeronikaEngler is this format following a certain style guide you remember? in that case, we could nicely use an existing plugin to citation.js for example.

**Comment by MauPalantir** [comment]
MLA is pretty close for example:
https://owl.purdue.edu/owl/research_and_citation/mla_style/mla_formatting_and_style_guide/mla_formatting_and_style_guide.html

**Comment by MauPalantir** [comment]
https://citation.js.org/

**Comment by dasch124** [comment]
Whenever there is a printed publication from `sourceDesc`, include this as well in the citation e.g.:

Naddari, A. (2025). Testour3/f/70 E 2 – Testour3. Based on: {printed source citation}. In V. Ritt-Benmimoun (Ed.), TUNOCENT - Tunisia’s Linguistic terra incognita: An Investigation into the Arabic Varieties of Northwestern and Central Tunisia. [Available online at https://tunocent-5b7ead81-4e64-4879-83de-7120e271b4f4.acdh-ch-dev.oeaw.ac.at/. Accessed on 2025-01-27.]

**Comment by dasch124** [comment]
Order of contributors:

first transcriber, then others in alphabetical order

NB
 
* Maybe needs revisiting, since corpus texts work differently than other
* maybe add explatory text "how to cite this web site"

**Comment by dasch124** [comment]
Dear @MauPalantir we have talked about this in our last meeting, could you please see if you can implement this until our next one?

**Comment by dasch124** [comment]
Agreed to include the full citation at the end of the block, before the square brackets. 

e.g.: Ritt-Benmimoun, V. (2025). SidiBouzid2/f/41 E 2 – SidiBouzid2. In V. Ritt-Benmimoun (Ed.), TUNOCENT - Tunisia’s Linguistic terra incognita: An Investigation into the Arabic Varieties of Northwestern and Central Tunisia. Based on: Ritt-Benmimoun, V. (2025). The Dialect Spoken in Sidi Bouzid: A Ghoul-Story from Central Tunisia. [Available online at URL. Accessed on 2025-05-09.]

**Comment by ctot-nondef** [comment]
@dasch124 could you check the tunocent dev instance and let us know what is still missing to close this? thank you!
